tests/log_parser/test_store_failure_lines.py

```diff
@@ -373,7 +373,6 @@ def test_confirm_failure_no_intermittent(
     ]
     jobs = create_errorsummary_job(hundred_job_blobs[0], create_jobs, log_filenames)
     job_logs = JobLog.objects.filter(job_id__in=(j.id for j in jobs))
-    assert len(jobs) == len(log_filenames)
```

Comment:
Please don't remember this check even if it's an intermittent failure.

Reply:
added this back